app/javascript/packs/application/components/Tour.test.js

```diff
@@ -1,6 +1,6 @@
-import Tour from "./Tour";
+import { Tour } from "./Tour";
 import React from "react";
-import { mount } from "enzyme";
+import { shallow } from "enzyme";
```

Comment:
I could be persuaded that it's overkill, but I'd like to see a test covering the `<p>loading... please wait!</p>` logic.  I'm assuming that displaying that loading message is important for business reasons and therefore worth preventing regressions.

Reply:
This is not something that should block this PR.  It could come in its own PR later 💯

Reply:
good idea! will do :-D